pkgs/applications/editors/vim/plugins/non-generated/cord-nvim/default.nix

```diff
@@ -7,19 +7,19 @@
   vimUtils,
 }:
 let
-  version = "0-unstable-2024-12-17";
+  version = "2.3.2-unstable-2024-12-22";
```

Comment:
We should use latest release instead of latest commit

https://github.com/vyfor/cord.nvim/releases

Reply:
Yeah, did you want to create that PR or I can, if you don't.

Reply:
I think we can do it in this PR